### PR TITLE
Check if project file is correct when looking for project dir

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: project-helper.ts [hunk 1]
```diff
@@ -55,5 +56,15 @@ export class ProjectHelper implements IProjectHelper {
 		let sanitizedName = _.filter(appName.split(""), (c) => /[a-zA-Z0-9]/.test(c)).join("");
 		return sanitizedName;
 	}
+	
+	private isProjectFileCorrect(projectFilePath: string): boolean {
+		if(this.$staticConfig.CLIENT_NAME_KEY_IN_PROJECT_FILE) {
+			var fileContent = this.$fs.readJson(projectFilePath).wait();
```
Comment (teobugslayer) [inline project-helper.ts]: This code could throw, should we catch it here and return false, or let uplevel  code handle it?

File: project-helper.ts [hunk 2]
```diff
@@ -55,5 +56,15 @@ export class ProjectHelper implements IProjectHelper {
...
+			var clientSpecificData = fileContent[this.$staticConfig.CLIENT_NAME_KEY_IN_PROJECT_FILE];
+			return clientSpecificData !== null && clientSpecificData !== undefined;
```
Comment (teobugslayer) [inline project-helper.ts]: consider writing it shortly like

```JavaScript
return !!clientSpecificData
```

File: project-helper.ts (hunk 1 above)
Comment (Fatme) [inline project-helper.ts]: Actually when we use this function we are sure that the file exists - https://github.com/telerik/mobile-cli-lib/pull/329/files#diff-2fc55d29fa5bf8690e0a4ae30a71effdR25

Comment (teobugslayer) [inline project-helper.ts]: It exists, but the content may not be a JSON.

**Comment by teobugslayer** [comment]
:+1:

**Pull request merged**